packages/playwright-core/bin/reinstall_chrome_stable_linux.sh

```diff
@@ -7,27 +7,23 @@ if [[ $(arch) == "aarch64" ]]; then
   exit 1
 fi
 
-is_user_root () { [ "${EUID:-$(id -u)}" -eq 0 ]; }
-if is_user_root; then
-  maybesudo=""
-else
-  maybesudo="sudo"
-fi
-
-
 # 1. make sure to remove old stable if any.
 if dpkg --get-selections | grep -q "^google-chrome[[:space:]]*install$" >/dev/null; then
-  $maybesudo apt-get remove -y google-chrome
+  apt-get remove -y google-chrome
 fi
 
+# 2. Update apt lists (needed to install curl and chrome dependencies)
+apt-get update
+
+# 3. Install curl to download chrome
 if ! command -v curl >/dev/null; then
-  $maybesudo apt-get install -y curl
+  apt-get install -y curl
 fi
 
-# 2. download chrome stable from dl.google.com and install it.
+# 4. download chrome stable from dl.google.com and install it.
 cd /tmp
 curl -O https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-$maybesudo apt-get install -y ./google-chrome-stable_current_amd64.deb
+apt-get install -y ./google-chrome-stable_current_amd64.deb
```

Comment:
note: This requires an up-to-date apt cache since its installs chrome dependencies